viewer/vueapp/src/components/visualizations/Visualizations.vue

```diff
@@ -769,6 +772,40 @@ export default {
           }
         });
       }
+
+      // add business hours to graph if they exist
+      if (!this.$constants.MOLOCH_BUSINESS_DAY_START || !this.$constants.MOLOCH_BUSINESS_DAY_END) {
+        return;
+      }
+      let businessDays = this.$constants.MOLOCH_BUSINESS_DAYS.split(',');
```

Comment:
Safer if this was wrapped in a `if (this.$constants.MOLOCH_BUSINESS_DAY_START && this.$constants.MOLOCH_BUSINESS_DAY_END)` just in case code is added after this piece.

Reply:
Yeah, not a bad idea 👍 Maybe I should also make this it's own function and just call it here. That might be cleaner